python/fusion_config/query_site_search_all_pipeline.json

```diff
@@ -1,193 +1,52 @@
-        },
-        {
-          "key": "hl.snippets",
-          "value": "2",
-          "policy": "append"
-        },
-        {
-          "key": "hl.fl",
-          "value": "title content body subject question_txt",
-          "policy": "append"
-        },
-        {
-          "key": "group",
-          "value": "true",
-          "policy": "append"
-        },
-        {
-          "key": "group.field",
-          "value": "site_search_s",
-          "policy": "append"
-        },
-        {
-          "key": "group.limit",
-          "value": "20",
-          "policy": "append"
-        },
-        {
-          "key": "bq",
-          "value": "(productName:fusion)",
-          "policy": "append"
-        },
-        {
-          "key": "bq",
-          "value": "(productVersion:3.0)^10",
-          "policy": "append"
-        },
-        {
-          "key": "bq",
-          "value": "(productVersion:2.4)^5",
-          "policy": "append"
-        },
-        {
-          "key": "mm.autoRelax",
-          "value": "true",
-          "policy": "append"
-        }
-      ],
-      "skip": false,
-      "label": "set-params"
-    },
-    {
-      "type": "boosts",
-      "id": "couvfmvxjd41dzpvi",
-      "queryParam": "q",
-      "rules": [
-        {
-          "keyword": "download",
-          "mode": "match",
-          "field": "id",
-          "boosts": [
-            {
-              "value": "https://lucidworks.com/download/",
-              "boost": 100
-            }
-          ]
-        },
-        {
-          "keyword": "fusion",
-          "mode": "match",
-          "field": "id",
-          "boosts": [
-            {
-              "value": "https://doc.lucidworks.com/index.html",
-              "boost": 5
-            },
-            {
-              "value": "https://lucidworks.com/download/",
-              "boost": 100
-            }
-          ]
-        }
-      ],
-      "skip": false,
-      "label": "boosts"
-    },
-    {
-      "type": "solr-query",
-      "id": "2aibvx6g1szslqsemi",
-      "allowedRequestHandlers": [],
-      "httpMethod": "POST",
-      "allowFederatedSearch": false,
-      "skip": false,
-      "label": "solr-query"
-    }
-  ],
-  "properties": {}
+  "id" : "site-search-all",
+  "stages" : [ {
+    "type" : "run-query-pipeline",
+    "id" : "cy9aw8ouu7m0zr529",
+    "pipelineToRun" : "site-search-base",
+    "type" : "run-query-pipeline",
+    "skip" : false,
+    "label" : "run-query-pipeline"
+  }, {
+    "type" : "set-params",
+    "id" : "e1grsryvjza3e4s4i",
+    "params" : [ {
+      "key" : "group",
+      "value" : "true",
+      "policy" : "append"
+    }, {
+      "key" : "group.field",
+      "value" : "site_search_s",
+      "policy" : "append"
+    }, {
+      "key" : "group.limit",
+      "value" : "10",
+      "policy" : "append"
+    } ],
+    "type" : "set-params",
+    "skip" : false,
+    "label" : "QueryParams - Grouping"
+  }, {
+    "type" : "set-params",
```

Comment:
The Query Fields stage lets you put in qf and fl params, not sure how we'd shoehorn in these grouping params.